### DISCUSSION TITLE
Allow infinity scroll

### DISCUSSION THREAD

**Discussion by Barbapapazes** [unpoly/unpoly#452] [Q&A]
Hello, 

Love this library but one thing is missing and it's infinity scroll.

Is there any way to enable this?

**Comment by adam12** [comment]
This will be heavily dependent on the way HTML is generated, so I don't think Unpoly could support this out of the box. But a few years ago I provided an example of how I would approach it with a small Unpoly compiler. Maybe it will be helpful to you as a starting point. https://github.com/unpoly/unpoly/issues/34
